native/packages/react-native-bpk-component-card/src/BpkCard.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  View,
+  Platform,
+  StyleSheet,
+  TouchableHighlight,
+} from 'react-native';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const tokens = Platform.select({
+  ios: () => require('bpk-tokens/tokens/ios/base.react.native.common.js'), // eslint-disable-line global-require
+  android: () => require('bpk-tokens/tokens/android/base.react.native.common.js'), // eslint-disable-line global-require
+})();
+
+const {
+  colorGray50,
+  colorGray900,
+  colorWhite,
+  spacingBase,
+} = tokens;
+
+/**
+ * Define styles needed by the component
+ */
+const styles = StyleSheet.create({
+  card: {
+    // TODO: update once tokens are available
+    backgroundColor: colorWhite,
+    elevation: 3,
+    shadowColor: colorGray900,
+    shadowOffset: { width: 0, height: 1 },
+    shadowOpacity: 0.3,
+    shadowRadius: 3,
+  },
+  common: {
+    // TODO: update once tokens are available
+    borderRadius: 8,
+  },
+  overflowHidden: {
+    // TODO: update once tokens are available
+    overflow: 'hidden',
+  },
+  padded: {
+    padding: spacingBase,
+  },
+});
+
+const BpkCard = (props) => {
+  const { padded, children, onPress, accessibilityLabel, ...rest } = props;
+
+  const cardStyle = [styles.card, styles.common];
+
+  if (padded) {
+    cardStyle.push(styles.padded);
+  }
+
+  return (
+
+    <TouchableHighlight
+      accessibilityComponentType="button"
+      accessibilityLabel={accessibilityLabel}
+      onPress={onPress}
+      style={cardStyle}
+      underlayColor={colorGray50}
+      {...rest}
+    >
+      <View
+        style={[styles.overflowHidden, styles.common]}
+      >
+        {children}
+      </View>
+    </TouchableHighlight>
+  );
+};
+
+BpkCard.propTypes = {
+  accessibilityLabel: PropTypes.string,
```

Comment:
Strictly speaking, we are doing extra work here by accepting this prop and passing it through i.e. if we didn;t do this it would still be handled by the spread `{...rest}`. Im in favor of removing it - we can still have it in the readme if we want to promote it and can add a test for doing so.

Reply:
Good idea, I didn't think of the `...rest` argument when I suggested the card accept this prop.

Reply:
I totally agree with that, we had a discussion yesterday about it and the decision was to put it, if @matthewdavidson is in favour of removing it I'll get rid of it and add a section in the readme for accessibility